src/server/apiVideo/index.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+const Request = require('./request');
```

Comment:
If we pass http client to a hook, as a service, we'd reduce repetition in the elements and have more control of what we want to expose in the future. 🤔 Thoughts?

Reply:
I like that idea. Should I include that in this PR?